python-client/wmill/wmill/client.py

```diff
@@ -23,7 +23,7 @@
 
 
 class Windmill:
-    def __init__(self, base_url=None, token=None, workspace=None):
+    def __init__(self, base_url=None, token=None, workspace=None, verify=True):
```

Comment:
httpx default is `True` -> https://github.com/encode/httpx/blob/08eff926a6011ac84961fd2cee22d9fd4d4b1a88/httpx/_client.py#L629